app/grants/sync/zcash.py

```diff
@@ -73,6 +77,19 @@ def get_zcash_txn_status(txnid):
     return None
 
 
+def is_txn_done_recently(time_of_txn):
```

Comment:
this check ensures that the txn was signed within 5 hours of submission on the Contribution record being created